velox/dwio/common/CachedBufferedInput.cpp

```diff
@@ -24,10 +24,8 @@ DEFINE_int32(
     80,
     "Minimum percentage of actual uses over references to a column for prefetching. No prefetch if > 100");
 
-DEFINE_int64(
-    max_coalesced_bytes,
-    128 << 20,
-    "Maximum size of single coalesced IO");
+DECLARE_int32(max_coalesced_distance_bytes);
+DECLARE_int64(max_coalesced_bytes);
```

Comment:
Are these needed at all or can they be deleted?

Reply:
Removed.